deploy/eck-operator/values.yaml

```
@@ -68,6 +68,12 @@ tolerations: []
 # affinity defines the node affinity rules for the operator pod.
 affinity: {}
 
+# Only one of minAvailable or maxUnavailable can be set
+podDisruptionBudget:
+  enabled: false
+  # minAvailable: 1
```

I would set by default `enabled: false` and `minAvailable: 1` (not commented).
```suggestion
  minAvailable: 1
```